staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -546,16 +682,13 @@ func watchHandler(start time.Time,
 	name string,
 	expectedTypeName string,
 	setLastSyncResourceVersion func(string),
+	exitOnInitEventBookmark *bool,
```

Comment:
nit: I'm sorry - that was my mental shortcut, I think we should name it
"exitOnInitialEventsEndBookmark" 
(for consistency with annotation name)

Reply:
np